components/Form/Button.js

```diff
@@ -167,10 +167,11 @@ const styles = StyleSheet.create({
     border: `1px solid ${colors.NEW_BLUE(1)}`,
     color: colors.NEW_BLUE(1),
     ":hover": {
-      borderColor: "#FFF",
-      boxShadow: "2px 2x 2px 2px #EDEDED",
-      backgroundColor: colors.NEW_BLUE(1),
-      color: "#FFF",
+      opacity: 0.6,
```

Comment:
Is this okay to comment out globally?

Reply:
Pretty sure it is -- the resulting button is pretty ugly